src/Components/test/E2ETest/ServerExecutionTests/RemoteRendererBufferLimitTest.cs

```diff
@@ -75,48 +53,11 @@ public async Task DispatchedEventsWillKeepBeingProcessed_ButUpdatedWillBeDelayed
             Client.ConfirmRenderBatch = true;
 
             // This will resume the render batches.
-            await Client.ExpectRenderBatch(() => Client.ConfirmBatch(Batches[^1].Id));
+            await Client.ExpectRenderBatch(() => Client.ConfirmBatch(Batches.Last().Id));
```

Comment:
Not a friend of `^1`?

Reply:
It's not supported with `IReadOnlyCollection<>` I think this only worked with `List<>` but that's not thread safe.